guardian/cdk-cli
# Per command args

## What does this change?

This PR refactors the way that arguments are handled to be on a per command basis. This paves the way for adding more commands which take different arguments which maintaining cross command utility functions. 

## How to test

Run the migrate command on an existing stack and ensure it still works as expected.

## How can we measure success?

Functionality is maintained while improving the project architecture to allow new commands to be added easily.

## Have we considered potential risks?

n/a

## Images

n/a
